apps/hash-frontend/src/pages/[shortname]/[page-slug].page.tsx

```diff
@@ -200,29 +194,22 @@ export const getServerSideProps: GetServerSideProps<PageProps> = async ({
   return {
     props: {
       pageWorkspace,
-      blocks: fetchedBlocks,
+      blocks: [],
```

Comment:
Does this mean that `UserBlocksProvider` is populated client side in a child component? Or is the value no longer populated at all?

Reply:
There's an effect inside `UserBlocksProvider` that fetches the list of blocks from the BP API and adds it to the initial list.

Since the server-side code was fetching default blocks (paragraph etc) which will appear in the BP API anyway, they will be fetched client-side in this effect.

I did notice while getting the link for this that it was making requests for block metadata serially. I've updated this in dba74666218679ed183e1cf5860f1e32b1e8a51a